DigiMeSDK/GuestConsent/Classes/PreConsentViewController.m

```diff
@@ -87,4 +102,13 @@ - (void)guestConsentClicked
     }
 }
 
+- (void)viewWillTransitionToSize:(CGSize)size withTransitionCoordinator:(id<UIViewControllerTransitionCoordinator>)coordinator
+{
+    [coordinator animateAlongsideTransition:^(id<UIViewControllerTransitionCoordinatorContext> context)
+     {
+         [self buildUiAndRotate:YES];
```

Comment:
I am a bit confused by the purpose of passing rotate boolean to a UI Setup method. if we need to rebuild UI I would rather split into 2 methods - 1 that resets, and another that builds again.

EG:
```objective-c
- (void)teardownUI
{
        [self.blurEffectView removeFromSuperview];
        [self.guestLabel removeFromSuperview];
        [self.preConsentView removeFromSuperview];
}

// then in animateAlongsideTransition:

[self teardownUI];
[self buildUI];
```